src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -369,7 +369,12 @@ public virtual Collection<PSObject> RunPowerShellTest(params string[] scripts)
                     if (powershell.Streams.Error.Count > 0)
                     {
                         throw new RuntimeException(
-                            "Test failed due to a non-empty error stream, check the error stream in the test log for more details.");
+                            string.Format(
+                                "Test failed due to a non-empty error stream. First error: {0}{1}",
+                                PowerShellUtilities.FormatErrorRecord(powershell.Streams.Error[0]),
```

Comment:
@jaredmoo either create a new local function in this file or simply inline the format wrapper function here.
It's little odd to call a function inside string.format() that basically calls string.format()
But if you feel strong to have a separate utility function for string.format, please declare locally where you have found the need to call it.
Else please let me know the scenarios where you will need the same function to be called from anywhere else.